src/jquery.autofilter.js

```diff
@@ -147,10 +177,12 @@
                             });
 
                             updateRowVisibility(table, allColumnFilterSets);
+                            $inputText.val('');
+                            $this.trigger('autofilter.FILTERING_END');
```

Comment:
What uses this event?

Reply:
I think it's useful to know when the filtering is done.
In my case, i wanted to do some fixes to the CSS once the filtering is done.